Views/EditorTemplates/Parts/AdvancedSKUProduct.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model Nwazet.Commerce.ViewModels.AdvancedSKUProductEditorViewModel
+
+@Html.HiddenFor(m => m.Product.Sku)
+
+@{
+    string skuHtml = "";
+}
+@if (Model.Settings.AllowCustomPattern) {
```

Comment:
This is only one of the reasons why I should not be allowed to do the views of things. Anyway, fixing this.